src/utils/shared/expiration.js

```diff
@@ -0,0 +1,56 @@
+const moment = require('moment')
+
+const DEFAULT_EXPIRATION_OFFSET = { days: 7 }
+
+/*
+   Get the expiration date for an object with date and/or expires fields
+   If expires is given, return it as a moment
+   If expires is undefined, return date plus 7 days
+   If expires is false, return false, as the item never expires.
+*/
+function getExpirationDate({ date, expires }) {
```

Comment:
I think we can add tests for this

Reply:
the function can be simplified I think ... try to check CodeClimate for cases like this

if (expires === true) return moment(expires)
if (expires === false) return null
if (date) return moment(date).add(DEFAULT_EXPIRATION_OFFSET)
return null

Reply:
Now that I fixed builds, tests are next on my radar.

I believe this is the simplest way to write the exact desired behavior for expiration dates and it seems to work, but automated tests will make that much easier to confirm.

Reply:
Code is simplified, and now there's tests that all pass!